mappings/net/minecraft/advancement/AdvancementProgress.mapping

```diff
@@ -1,25 +1,38 @@
 CLASS net/minecraft/unmapped/C_rpzbknxp net/minecraft/advancement/AdvancementProgress
+	FIELD f_mhfgclex OBTAINED_TIME_FORMAT Ljava/time/format/DateTimeFormatter;
```

Comment:
`achieved`?

Reply:
Hm, achieved is nice because it's corresponds to achievement which is like advancement...
This works well for the the members that refer to the entire `AdvancementProgress` like this field and `isComplete/Obtained` with no args...
However I don't think it makes as much sense when talking about `Criterion`s.

I guess it's fine to use `achieved` for `Progress` as a whole and use `complete` for `Criterion`; it could even make make things clearer.

Reply:
I think that unifying under `complete` is nicer, as a personal preference. I like consistency